etc/kayobe/inventory/group_vars/compute/firewall

```diff
@@ -47,6 +47,11 @@ firewallgen_ipv4_input_allow_rules:
     proto: tcp
     destination: "{{ internal_net_name | net_ip }}"
     comment: "hint: used by 'mtail' in docker container 'prometheus_mtail'"
+  - interface: "{{ internal_net_name | net_interface }}"
+    port: 10000:20000
```

Comment:
I could have swarn will got this working once, but its sure broken without this.

Looks like the expected range.
https://github.com/openstack/nova/blob/c883f2201026b9237a1fba9a0a55ff607d57e787/nova/conf/serial_console.py#L18